README.md

```diff
@@ -37,6 +41,31 @@ b = Dagger.@spawn rand(a, 4)
 c = Dagger.@spawn sum(b)
 fetch(c) # some number!
 ```
+## Contributing Guide
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+![GitHub issues](https://img.shields.io/github/issues/JuliaParallel/Dagger.jl)
+![GitHub contributors](https://img.shields.io/github/contributors/JuliaParallel/Dagger.jl)
+[![GitHub](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com)
```

Comment:
```suggestion
```
I don't think we need a GitHub badge, unless you have a particular reason you think it should be here?